data/mods/ssb/statuses.js

```diff
@@ -475,6 +475,19 @@ let BattleStatuses = {
 			this.add(`c|&false|ɢ∞פ⋆ᖙᵒᵝ ٩꒰”̮*ू꒱`);
 		},
 	},
+	fart: {
+		noCopy: true,
+		onStart(target, source) {
+			source.types = ['Fairy', 'Steel'];
```

Comment:
You are gonna need to rework this, see the pokedex.js file and OM's status for how to do this

Reply:
Weird. I had it that way at first but fsr I thought I had to change it here.

Reply:
Ah, it was because when I did it how OM had it, it wasn't displaying the type change.

Reply:
When the type is changed in pokedex.js and here, it still shows Kartana as Grass/Steel. OM's I think is different because it doesn't change Flareon's base typing. The typeadd won't work because the signature move also makes use of that.

Reply:
i see

Reply:
Ok so keep the code in pokedex.js and add this line: 
```suggestion
			this.add('-start', source, 'typechange', `Fairy/Steel}`);
```